helm_deploy/hmpps-restricted-patients/values.yaml

```diff
@@ -43,6 +43,8 @@ generic-service:
       SYSTEM_CLIENT_ID: "SYSTEM_CLIENT_ID"
       SYSTEM_CLIENT_SECRET: "SYSTEM_CLIENT_SECRET"
       SESSION_SECRET: "SESSION_SECRET"
+      TAG_MANAGER_CONTAINER_ID: "TAG_MANAGER_CONTAINER_ID"
+      GOOGLE_ANALYTICS_ID: "GOOGLE_ANALYTICS_ID"
```

Comment:
I guess that these will need to be put in the dev environment (i.e. added to the secrets?)